example/panner/mainStereo.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+// Create an instance
+var wavesurfer = Object.create(WaveSurfer);
+
+// Init & load audio file
+document.addEventListener('DOMContentLoaded', function () {
+    // Init
+    wavesurfer.init({
+        container: document.querySelector('#waveform'),
+        minPxPerSec: 30,
+        scrollParent: true,
+        waveColor: '#A8DBA8',
+        progressColor: '#3B8686'
+    });
+
+    // Load audio from URL
+    wavesurfer.load('../split-channels/stereo.mp3');
+    
+    // Panner
+    (function () {
+        // Add panner
+        wavesurfer.splitter = wavesurfer.backend.ac.createChannelSplitter();
+        wavesurfer.merger = wavesurfer.backend.ac.createChannelMerger();
+        wavesurfer.channelVolumeNode = [];
+
+        wavesurfer.backend.analyser.disconnect();
+
+        wavesurfer.backend.analyser.connect(wavesurfer.splitter);
+        for (var channel = 0; channel < wavesurfer.backend.ac.destination.channelCount; channel++) {
+			wavesurfer.channelVolumeNode.push(wavesurfer.backend.ac.createGain());
+			wavesurfer.splitter.connect(wavesurfer.channelVolumeNode[channel],channel);
+			wavesurfer.channelVolumeNode[channel].connect(wavesurfer.merger,0,channel);
+		};
+        wavesurfer.backend.connect(wavesurfer.merger,wavesurfer.backend.gainNode,wavesurfer.backend.ac.destination,'series');
```

Comment:
This whole section above is horribly lengthy for connection. But it seems that this is only the case because I've to use the channelMerger and channelSplitter nodes. It'll look much cleaner after I use the connectionCore method for the connection between the splitter and merger.